packages/core/src/hooks/usePlate/useEditableProps.ts

```diff
@@ -15,19 +15,20 @@ export const useEditableProps = ({
   id = 'main',
 }: Pick<PlateProps, 'id'>): EditableProps => {
   const editor = usePlateEditorRef(id);
-  const keyPlugins = usePlateSelectors(id).keyPlugins();
+  const plugins = usePlateSelectors(id).plugins();
```

Comment:
@zbeyens I noticed that `keyPlugins` isn't changing the way it's supposed to so I used `plugins` and that made it "work". I'm going to look into why `keyPlugins` isn't changing.

Reply:
FYI `keyPlugins` is a number incremented each time `plugins` change

Reply:
Thanks for the insight, that's helpful to figuring out where to look next!

That's what I thought it was supposed to do (I looked through the store code a little) but since it's not updating on plugin change (logging it in a useEffect didn't show it changing), my next exploration is to see what's causing that.

Reply:
@zbeyens I stand corrected, it's possible that I didn't give the dev server time to reload when my `useEffect` console.log wasn't logging `keyPlugins` changed. 

It turned out that it's because the `isValid` value wasn't memo'ized, so it was never changing for the `useMemo` calls that were using it. I left more a detailed message in my PR comment about why I did it the way I did, I'm happy to go a different route if removing the `isValid` variable and instead passing `keyPlugins` into the `useMemo` calls as a dependency isn't the best approach.